javascript-if/main.js

```diff
@@ -0,0 +1,56 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === string[0].toUpperCase()) {
```

Comment:
so this is checking if the first letter is uppercase, instead, you want to check if the first letter is a capital `J`